### DISCUSSION TITLE
'Hackweek-avatar-maker' integrated on 'hubs cloud'

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#4408] [Avatar Questions]
I want to know how to 'https://github.com/mozilla/hackweek-avatar-maker' avatar code and 'hubs cloud' tied together?Maybe integrating code in the 'hubs cloud';Maybe on the server of 'hubs cloud'.

**Comment by matthewbcool** [comment]
Hi LiHong! The two are not tied together AFAIK.  I could see how it might be valuable to integrate it with the client but there are no guides to help with that at the moment.

**Reply by lihong-xmuseum** [reply]
Thank you very much for telling me. So, is it up to me to find my own way?
